hyperspy/_components/bleasdale.py

```diff
@@ -42,18 +42,47 @@ class Bleasdale(Expression):
             Extra keyword arguments are passed to the ``Expression`` component.
         
     
-    For :math:`(a+b\cdot x)\leq0`, the component will be set to nan.
-
+    For :math:`(a+b\cdot x)\leq0`, the component will be set to 0.
     """
     
-    def __init__(self, a=1., b=1., c=1., module="numexpr", **kwargs):
+    def __init__(self, a=1., b=1., c=1., module="numexpr",
+                 compute_gradients=False, **kwargs):
```

Comment:
Since sympy cannot compute the gradients in this case, wouldn't it be better not to provide the option to enable them here?

Reply:
done